src/node/chain.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 The Bitcoin Core developers
```

Comment:
Maybe add this new file to iwyu?

Reply:
https://cirrus-ci.com/task/4504627910541312?logs=ci#L6998

Reply:
Looks like an iwyu bug, so maybe undo for now?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25494#discussion_r921900387

> https://cirrus-ci.com/task/4504627910541312?logs=ci#L6998

> Looks like an iwyu bug, so maybe undo for now?

I'm not actually sure what the bug is but I added some missing includes and undid change that adds the file to IWYU for now. The larger PR https://github.com/bitcoin/bitcoin/pull/24230 this was part of adds more to this file so there is a chance to enable IWYU again later